src/bench/bench_bitcoin.cpp

```diff
@@ -30,6 +30,9 @@ static void SetupBenchArgs(ArgsManager& argsman)
     argsman.AddArg("-output-csv=<output.csv>", "Generate CSV file with the most important benchmark results", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     argsman.AddArg("-output-json=<output.json>", "Generate JSON file with all benchmark results", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
     argsman.AddArg("-sanity-check", "Run benchmarks for only one iteration", ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
+    argsman.AddArg("-priority-level=<l1,l2,l3>", strprintf("Run benchmarks of one or multiple priority level(s) (%s | %s | %s), default: '%s'", PriorityToString(benchmark::PriorityLevel::LOW),
+                                                           PriorityToString(benchmark::PriorityLevel::MEDIUM), PriorityToString(benchmark::PriorityLevel::HIGH),
+                                                           PriorityToString(benchmark::DEFAULT_PRIORITY_LEVEL)), ArgsManager::ALLOW_ANY, OptionsCategory::OPTIONS);
```

Comment:
I am not sure that I see the benefit of the `PriorityToString` function, I believe that it adds extra circular transformation complexity (priority -> string -> priority) without a solid benefit.

In case of a change to priorities (adding/removing options), we will still need to make a change here. Maybe a function that uses `map_priority_level` to concatenate all available options could be a better fit. With this changed what's left for `PriorityToString` is its usage with `DEFAULT_PRIORITY_LEVEL` which can be erased by using string instead of priority:`DEFAULT_PRIORITY_LEVEL{"all"}`.
(afc9c1bb544dfc5418102dd2e5f9f2819b24b885)

Reply:
sounds good, pushed inside 085e88ef.